packages/backend/src/links/links.service.ts

```diff
@@ -27,12 +27,29 @@ export class LinksService {
   }
 
   async findAll(params: SearchLink) {
+    if (typeof params.term === 'undefined') return this.prisma.link.findMany()
     return this.prisma.link.findMany({
       where: {
-        title: {
-          contains: params.title,
-          mode: 'insensitive',
-        },
+        OR: [
+          {
+            title: {
+              contains: params.term,
+              mode: 'insensitive',
+            },
+          },
+          {
+            slug: {
```

Comment:
One more thing, based on the user story @Tschonti had written, this should not be slug, it should be "keywords" instead right?

Reply:
I added both of them now.